### ISSUE TITLE
IREE roadmap

### ISSUE THREAD

**Issue by denis0x0D** [google/iree#158]
Hello, can you please clarify on this
> Our goal for the end of the year is to have depth in a few complex examples (such as streaming speech recognition) and breadth in platforms. This should hopefully allow for contributions both from Googlers and externally to enable broader platform support and optimizations as well as prove out some of the core IREE concepts.

What kind of models you are planning to be able to run to the end of this year?
Thanks.

**Comment by stellaraccident** [comment]
Aside from mnist and other toy samples, we are focusing most of our energy on a variety of architectures from the speech and nlp spaces: various LSTM-based networks, Transformers, svdf (for keyword spotting), etc (we're focusing on a couple of specific ones that have non-trivial/algorithmic decoders). We have found that current tooling is somewhat limited for these classes, and that is where we chose to start. I expect that we will also have the basics for running some classes of convolutional vision networks, but that is not what we are focusing on first (tooling for these is already quite advanced and we would rather compete on these dimensions once we have more of the foundations in place).

Several of us working on IREE have historically had alignment with the work that is done as part of the TensorFlow lingvo project, and we are drawing from some of those implementations to bootstrap the work.
